spec/integration/authorization/pundit_spec.rb

```diff
@@ -0,0 +1,228 @@
+require 'spec_helper'
+
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def new?
+    user.roles.include? :admin
+  end
+
+  def show?
+    true
+  end
+
+  def update?
+    user.roles.include? :admin
+  end
+
+  def create?
+    user.roles.include? :admin
+  end
+
+  def edit?
+    user.roles.include? :admin
+  end
+
+  def destroy?
+    user.roles.include? :admin
+  end
+
+  def rails_admin?(action)
+    case action
+    when :dashboard
+      user.roles.include? :admin
+    when :index
+      false
+    when :show
+      user.roles.include? :admin
+    when :new
+      user.roles.include? :admin
+    when :edit
+      user.roles.include? :admin
+    when :destroy
+      false
+    when :export
+      user.roles.include? :admin
+    when :history
+      user.roles.include? :admin
+    when :show_in_app
+      user.roles.include? :admin
+    else
+      fail ::Pundit::NotDefinedError.new("unable to find policy #{action} for #{record}.")
+    end
+  end
+end
+
+class PlayerPolicy < ApplicationPolicy
+  def rails_admin?(action)
+    case action
+    when :index
+      user.roles.include? :admin
+    when :show
+      true
+    when :new
+      (user.roles.include?(:create_player) || user.roles.include?(:admin) || user.roles.include?(:manage_player))
+    when :edit
+      (user.roles.include? :manage_player)
+    when :destroy
+      (user.roles.include? :manage_player)
+    when :export
+      user.roles.include? :admin
+    when :history
+      user.roles.include? :admin
+    when :show_in_app
+      (user.roles.include?(:admin) || user.roles.include?(:manage_player))
+    else
+      fail ::Pundit::NotDefinedError.new("unable to find policy #{action} for #{record}.")
+    end
+  end
+end
+
+describe PlayerPolicy do
```

Comment:
We don't test behavior of policies themselves here.
What we want to test is integration of RailsAdmin and Pundit, so ensuring pundit's functionality or policy's correctness is not our job.